src/subdomains/supporting/bank/bank-tx/bank-tx.service.ts

```diff
@@ -34,11 +35,17 @@ export class BankTxService {
     private readonly olkyService: OlkypayService,
     private readonly bankTxReturnService: BankTxReturnService,
     private readonly bankTxRepeatService: BankTxRepeatService,
+    private readonly buyService: BuyService,
   ) {}
 
   // --- TRANSACTION HANDLING --- //
   @Cron(CronExpression.EVERY_MINUTE)
```

Comment:
Decrease to every 30s (requested by @DFXAdmin 😉)

Reply:
Bisch de bescht 😀